content/en/docs/distributions/gke/deploy/deploy-cli.md

```diff
@@ -187,7 +187,7 @@ Follow these steps to verify the deployment:
    your `kubectl` credentials to point to the new cluster:
 
     ```bash
-    gcloud container clusters get-credentials "${KF_NAME}" --zone "${ZONE}" --project "${KF_PROJECT}"
+    gcloud container clusters get-credentials "${KF_NAME}" --zone "${REGION}" --project "${KF_PROJECT}"
```

Comment:
Since the flag is `--zone`, we should use `${ZONE}` for this flag.

Reply:
https://cloud.google.com/sdk/gcloud/reference/container/clusters/get-credentials

Reply:
Fixed